Podfile

```diff
@@ -143,7 +143,7 @@ abstract_target 'Apps' do
 
   pod 'NSURL+IDN', '~> 0.4'
 
-  pod 'WPMediaPicker', '~> 1.8.8'
+  pod 'WPMediaPicker', '~> 1.8.9-beta.1'
```

Comment:
We have check that warn us when shipping a code freeze with a beta, but not at the time of release finalization (unless I missed/ignored them).

No big deal, but something to keep in mind.

For reference: https://github.com/wordpress-mobile/MediaPicker-iOS/releases/tag/1.8.9-beta.1

Reply:
In case you plan to work on that soon (i.e. add the similar check than the one in code_freeze to be also done on the release finalization), I wonder if that couldn't be an occasion to update the check to actually "refuse" any `-beta` tag, and let developer know that we decided to stop doing intermediate betas for our internal pods, and instead they could point to the commit sha during development]? (paaHJt-4lp-p2) 🤔